### PR TITLE
cb192 minors4

### PR DESCRIPTION
**Pull request by Brandonz07**
image

### CODE REVIEW

**Review by Angelbt12 — APPROVED** [review]
meets the requirements

File: src/pages/board/outlets/financialReporting/ToDo/index.tsx [hunk 1]
```diff
@@ -181,15 +181,16 @@ function ToDo(props: ToDoProps) {
...
                 options={toDo?.[0]?.decisions ?? []}
                 onChange={onChangeDecision}
                 disabled={toDo === undefined}
               />
             </Stack>
             <Stack
               padding="16px 0px 0px 0px"
-              width={isMobile ? "100%" : "auto"}
+              width="100%"
+              justifyContent="end"
```
Comment (carin91) [inline src/pages/board/outlets/financialReporting/ToDo/index.tsx:192]: Relocate to send button to its original position:
image

File: src/pages/board/outlets/financialReporting/ToDo/index.tsx [hunk 2]
```diff
@@ -198,6 +199,7 @@ function ToDo(props: ToDoProps) {
                 loading={button?.loading || false}
                 type="submit"
                 fullwidth={isMobile}
+                spacing="compact"
               >
                 {button?.label || "Enviar"}
```
Comment (carin91) [inline src/pages/board/outlets/financialReporting/ToDo/index.tsx:204]: Every spanish text must be located in a config file

File: src/pages/board/outlets/financialReporting/ToDo/index.tsx [hunk 3]
```diff
@@ -238,13 +241,14 @@ function ToDo(props: ToDoProps) {
               value={assignedStaff.analyst}
               fullwidth
               disabled={staff === null}
+              size="compact"
             />
             {icon && !isMobile && (
```
Comment (carin91) [inline src/pages/board/outlets/financialReporting/ToDo/index.tsx:246]: Aligned the icon with the analyst and commercial text fields. Its current location seems invalid.
image

File: src/pages/board/outlets/financialReporting/ToDo/index.tsx [hunk 4]
```diff
@@ -198,6 +199,7 @@ function ToDo(props: ToDoProps) {
                 loading={button?.loading || false}
```
Comment (carin91) [inline src/pages/board/outlets/financialReporting/ToDo/index.tsx:199]: image
aligned the send button with the text field

**Review by carin91 — CHANGES_REQUESTED** [review]

**Review by carin91 — APPROVED** [review]
image
Adjust the error

**Pull request merged**